File: lingting-spring-huawei/src/test/java/live/lingting/spring/huawei/SpringHuaweiApplication.java
package live.lingting.spring.huawei;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.ImportAutoConfiguration;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/**
 * @author lingting 2024-09-18 20:57
 */
@SpringBootApplication
@ImportAutoConfiguration({ HuaweiAutoConfiguration.class })
public class SpringHuaweiApplication {

	public static void main(String[] args) {
		SpringApplication.run(SpringHuaweiApplication.class, args);
	}

}

File: lingting-spring-huawei/src/test/kotlin/live/lingting/spring/huawei/SpringHuaweiApplication.kt
package live.lingting.spring.huawei

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.ImportAutoConfiguration
import org.springframework.boot.autoconfigure.SpringBootApplication

/**
 * @author lingting 2024-09-18 20:57
 */
@SpringBootApplication
@ImportAutoConfiguration(HuaweiAutoConfiguration::class)
object SpringHuaweiApplication {
    @JvmStatic
    fun main(args: Array<String>) {
        SpringApplication.run(SpringHuaweiApplication::class.java, *args)
    }
}
